### PR TITLE
Update strings.po all languages

### PR DESCRIPTION
**Pull request by icarok99**

### CODE REVIEW

**Comment by i96751414** [comment]
Thank you. I will have a look as soon as I have some spare time.

File: resources/language/resource.language.pt_pt/strings.po [hunk 1]
```diff
@@ -28,7 +28,7 @@ msgstr "Ativar cache"
 
 msgctxt "#30002"
 msgid "Cache expiration time (days)"
-msgstr "Tempo de expiração da cache (dias)"
+msgstr "Tempo de expiração do cache (dias)"
```
Comment (i96751414) [inline resources/language/resource.language.pt_pt/strings.po]: In PT-PT its: cache is female :p
The previous string was correct

File: resources/language/resource.language.pt_pt/strings.po [hunk 2]
```diff
@@ -56,7 +56,7 @@ msgstr "Tempo limite de resolução dos provedores (segundos)"
 
 msgctxt "#30009"
 msgid "Username"
-msgstr "Nome de Utilizador"
+msgstr "Nome do usuário"
```
Comment (i96751414) [inline resources/language/resource.language.pt_pt/strings.po]: PT-PT typically says "utilizador" and not "usuário", please revert.

File: resources/language/resource.language.pt_pt/strings.po [hunk 3]
```diff
@@ -185,7 +185,7 @@ msgstr "Por favor, aguarde..."
 
 msgctxt "#30111"
 msgid "Getting results from providers..."
-msgstr "A obter resultados dos provedores..."
+msgstr "Obtendo resultados dos provedores..."
```
Comment (i96751414) [inline resources/language/resource.language.pt_pt/strings.po]: Please revert as the previous phrase is more common/used in PT-PT (we usually don't use gerund, specially when starting sentences)

File: resources/language/resource.language.pt_pt/strings.po [hunk 4]
```diff
@@ -301,15 +301,15 @@ msgstr "Reproduzir"
 
 msgctxt "#30140"
 msgid "Adding contents to library"
-msgstr "A adcionar conteúdos na biblioteca"
+msgstr "Adicionando conteúdos à biblioteca"
```
Comment (i96751414) [inline resources/language/resource.language.pt_pt/strings.po]: Please revert as the previous phrase is more common/used in PT-PT (we usually don't use gerund, specially when starting sentences)

File: resources/language/resource.language.pt_pt/strings.po [hunk 5]
```diff
@@ -301,15 +301,15 @@ msgstr "Reproduzir"
...
 
 msgctxt "#30141"
 msgid "Updating library"
-msgstr "A atualizar biblioteca"
+msgstr "Atualizando a biblioteca"
```
Comment (i96751414) [inline resources/language/resource.language.pt_pt/strings.po]: Please revert as the previous phrase is more common/used in PT-PT (we usually don't use gerund, specially when starting sentences)

File: resources/language/resource.language.pt_pt/strings.po [hunk 6]
```diff
@@ -301,15 +301,15 @@ msgstr "Reproduzir"
...
 
 msgctxt "#30142"
 msgid "Re-building library"
-msgstr "A reconstruir biblioteca"
+msgstr "Reconstruindo a biblioteca"
```
Comment (i96751414) [inline resources/language/resource.language.pt_pt/strings.po]: Please revert as the previous phrase is more common/used in PT-PT (we usually don't use gerund, specially when starting sentences)

**Review by i96751414 — CHANGES_REQUESTED** [review]
Thanks for the updates. I did a review, and PT-PT strings need update

**Comment by icarok99** [comment]
@i96751414 revestir

Essas foram às únicos que mudei

msgctxt "#30018"
msgid "Show unaired episodes"
msgstr "Exibir episódios não emitidos"

msgctxt "#30019"
msgid "Show special episodes"
msgstr "Exibir episódios especiais"

**Review by i96751414 — APPROVED** [review]

**Comment by i96751414** [comment]
Can you squash before I merge, please?

**Comment by icarok99** [comment]
> Can you squash before I merge, please?

It's you who has to do this by choosing the 2nd option.

**Comment by i96751414** [comment]
I think that by doing that I'll remove your name from the contributions as it assumes it was me doing things. But I'll give it a try

**Comment by icarok99** [comment]
Remove no 

Screenshot_2024-02-25-16-43-07-922_com android chrome

**Pull request merged**